app/javascript/mastodon/locales/en.json

```diff
@@ -356,6 +356,17 @@
   "home.pending_critical_update.link": "See updates",
   "home.pending_critical_update.title": "Critical security update available!",
   "home.show_announcements": "Show announcements",
+  "ignore_notifications_modal.disclaimer": "Mastodon cannot inform users that you've ignored their notifications. Ignoring notifications will not stop the messages themselves from being sent.",
+  "ignore_notifications_modal.filter_instead": "Filter instead",
+  "ignore_notifications_modal.filter_to_act_users": "Filtering helps avoid potential confusion",
```

Comment:
This seems like a duplicate text. The default message reads "You'll still be able to accept, reject, or report users".